### PR TITLE
[Boxstation] Robotics Remap

### PR DESCRIPTION
**Pull request by Steelpoint**
This PR is a split off my minor map change PR (https://github.com/tgstation/tgstation/pull/24502) and a continuation of my two year old Robotics Remap PR.

In game image.
newrobo

Thank you for reading.

:cl:
add: Robotics and Mech Bay have seen a mapping overhaul on Boxstation.
add: A cautery surgical tool has been added to the Robotics surgical area on Boxstation.
/:cl:

### CODE REVIEW

**Comment by ImFlavo** [comment]
I haven't been on box station in a hot minute but those R_walls look inconsistent. I could be wrong though.

**Comment by Steelpoint** [comment]
Which rwalls?

**Comment by GunHog** [comment]
My previous comment, which is still relevant to this discussion has been posted below for your review. This feedback also includes Shaps' concern.

>You are missing a fuel tank, two holopads (One for the mech bay, another for robotics lab), and the AI display within the room. The southern portion of the mech bay used to be R-walls, but that section is now part of the robotics lab, I guess. Did you decide to get rid of the windoor for the surgery room? I would have liked to keep it. Anyone who does not like having to open it can just click it to keep it open forever.

>Another note, some of your items (Flashes, metals, etc) do not seem well sorted. Maybe add a table next to the console and move some things there so you do not have several mixed piles. The circuit imprinter needs to be kept clear! Players need to click there to get the boards, and those items are in the way.

Old Robotics below:
old robotics

**Comment by Steelpoint** [comment]
I've effected some minor changes, with a new image below.

More so I've slightly rearranaged the circuit imprinter, added two holopads and a surgical windoor.

newrobo

**Comment by OtherDalfite** [comment]
Looks good. Im glad surgery is sort of hidden away since roboticists often have little they can do for killing someone discretely.

**Comment by ExcessiveUseOfCobblestone** [comment]
Why does mech bay need rwalls when mechs have security measures of their own?

**Comment by GunHog** [comment]
Good stuff, Steelpoint!

**Comment by RemieRichards** [comment]
as I said on the other one.
make the surgery bit white please.

**Comment by GunHog** [comment]
Why white? If you did that, it would meld into the other white tiles and would no longer look as much as a dedicated sub-room. I rather like the darker shade in the room, it looks slightly sinister that way.

**Comment by Steelpoint** [comment]
Darker shade is nicer in my opinion, looks more like a backroom area.

**Comment by KorPhaeron** [comment]
This looks like it will be a lot of extra running back and forth

**Comment by Aliannera** [comment]
Old Box robotics always felt cramped and tight, hopefully this helps open it up a bit, as other stations often feel more open. Not a big fan of surgery being it's own little chamber but it's not a huge deal. Just means moving brains/limbs/augments might be a little annoying at times.

Love dat mech bay though.

**Comment by InsaneHyena** [comment]
Gonna repeat it. As a roboticist player, I don't like those changes.
1) It's too big for no reason
2) Surgery is isolated and too far, I won't be able to see front desk from the surgery table
3) Conveyor belts are silly and don't serve any actual purpose

**Comment by Steelpoint** [comment]
While I think its subjective, but I made the work area more 'open' because I personally more so prefer a open workspace to the more cramped workspace that we have currently.

I acknowledge that surgery is isolated, but this is part intentional, part necessary. You gain more privacy in your surgery, but you lose visibility.

The conveyor belts are the hill I'll die on to protect, it may be void of purpose but I think they are cool, and I like seeing the items being moved over to a work area.

**Comment by Dorsisdwarf** [comment]
I actually like the conveyor belts more than the rest of the PR, they are indeed cool.

**Comment by Dorsisdwarf** [comment]
Is there access restrictions on the weird windoor? why does it exist?

**Comment by Steelpoint** [comment]
Windoor has robotics access restrictions. Its there because I like it being there.

**Comment by KorPhaeron** [comment]
The windoor will just make moving patients back and forth annoying

**Comment by Steelpoint** [comment]
You can always click the windoor to keep it open forever

**Comment by GunHog** [comment]
It will also be a cool place to kill people and make it slightly harder for them to run! :dagger:

**Comment by Steelpoint** [comment]
Well travis is failing checks for a reason beyond my comprehension. 

May have to do a new PR at this rate.

File: tgstation.dme [hunk 1]
```diff
@@ -2042,4 +2042,5 @@
 #include "interface\interface.dm"
 #include "interface\stylesheet.dm"
 #include "interface\skin.dmf"
+#include "C:\Users\Steelpoint\Desktop\roboholdtemp.dmm"
```
Comment (Jordie0608) [inline tgstation.dme]: > tgstation.mdme:2048:error: unable to open "C:/Users/Steelpoint/Desktop/roboholdtemp.dmm".

Hmmmmm.

Comment (Steelpoint) [inline tgstation.dme]: Don't even want to know how that got there.

**Comment by ImFlavo** [comment]
qap42zw

For the most part entering robotics from outside requires breaking in via R_walls. except for these 2 walls where the arrows are pointing (ignore the arrow pointing at an R_wall)

**Comment by Robustin** [comment]
Neat, I will add a heart and confetti emoji if you can give me a quick rundown on mapmerging, i.e. which version of python to download, how not to make shit terrible, etc.

**Comment by Jordie0608** [comment]
@Robustin See https://tgstation13.org/wiki/Map_Merger

**Comment by Penguaro** [comment]
There is a problem with this map change. The Aft Robotics Fabricator is too far from the Robotics R&D Console to connect. An Issue was created for this.

**Pull request merged**